src/posting/collection.py

```diff
@@ -117,10 +117,28 @@ def to_httpx_args(self) -> dict[str, Any]:
         if self.content:
             httpx_args["content"] = self.content
         if self.form_data:
-            # Ensure we don't delete duplicate keys
-            httpx_args["data"] = tuples_to_dict(
-                [(item.name, item.value) for item in self.form_data if item.enabled]
-            )
+            files = []
+            data = defaultdict(list)
+
+            for item in self.form_data:
+                if not item.enabled:
+                    continue
+
+                if item.value.startswith("@"):
+                    file_path = item.value[1:]
+                    if not Path(file_path).exists():
```

Comment:
I'm undecided on whether the paths should be relative to the cwd or relative to the root of the collection directory. I'm concerned that if you were to share a collection, relative paths could break if they're cwd-relative.

Reply:
Based on my experience, developers rarely share files alongside collections, so I would suggest omitting the collection directory. Additionally, the current working directory (cwd) can vary between machines. What I’ve commonly seen from other clients is the use of a general working directory like ~/home/client_x. You can set that up and place all your files there for consistency.

That's not to say, you must always place your files there, you should always be able to set an absolute path.